<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S_DES</class>
 <widget class="QMainWindow" name="S_DES">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>S-DES算法</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="SNameLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>S-DES算法</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="GroupName">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>xxx组</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EncryptButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>加密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecrypthButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>470</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>解密</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>71</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="TextLabel">
       <property name="text">
        <string>字符串或8位二进制明密文：   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TextlineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>521</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="KeyLabel">
       <property name="text">
        <string>10位二进制原始密钥：        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="KeylineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="MessageBrowser">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>521</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
